internal/commands/pull/command.go

```diff
@@ -111,10 +111,9 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		return err
 	}
 
-	// TODO(REALMC-9462): maybe make this less hacky (pass app in to resolveClientTemplates perhaps?)
```

Comment:
can we actually remove this yet?  tbh i forget where we ultimately landed on this one, but the ticket itself is still in backlog

Reply:
I removed it from here because I don't think there's anything to do in this location for REALMC-9462.

It seems like we made a temporary app service method and use it here: https://github.com/10gen/realm-cli/blob/1436051fe79d09dc3c6e986562a70ce41ce4b3da/internal/commands/pull/inputs.go#L89

I think once we tackle REALMC-9462 (by adding template id into the /apps payload), we would ideally remove the code in the `resolveRemoteApp` and that'd be it